### DISCUSSION TITLE
Maven issue - skip over invalid config when using regex manager with datasource and packageName

### DISCUSSION THREAD

**Discussion by SaharItzhakWhiteSource** [renovatebot/renovate#26211] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### What is your question?

I use Renovate to get the latest package managers release by using the regexManager.
All other package-managers are working fine, but I get an error only on Maven.

Main script (Github-actions):
```
# renovate: datasource=maven packageName=org.apache.maven:maven
install-tool maven 3.5.4
```

renovate.json:
[renovate.json](https://github.com/renovatebot/renovate/files/13627124/renovate.json)

Mend logs:
![Renovate-Maven_Log](https://github.com/renovatebot/renovate/assets/83421212/3414c7c6-147f-4b78-957f-3b52f47ebf41)

Note: 
I've also tried changing to different packageNames, depNames but nothing seems to work.
I saw that this issue might be related but the fix is already merged - https://github.com/renovatebot/renovate/discussions/25758

### Logs (if relevant)

<details><summary>Logs</summary>

```

Copy/paste the relevant log(s) here, between the starting and ending backticks

```

</details>

**Comment by rarkins** [comment, marked as answer]
You have a rogue space after `maven`:

<img width="270" alt="image" src="https://github.com/renovatebot/renovate/assets/6311784/1379c630-6278-457a-8b51-e06b7f24bebc">
